cmake/compilers/Clang.cmake

```diff
@@ -12,6 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if(CMAKE_CXX_COMPILER MATCHES "/em\\+\\+(-[a-zA-Z0-9.])?$")
+  message(" * C++ compiler: Emscripten")
```

Comment:
This message may be useful while getting things working, but for consistency with the configuration of other compilers in oneTBB, it should probably be removed at some point.